spec/unit/recipes/mac_os_x_client_spec.rb

```diff
@@ -1,18 +1,25 @@
 require 'spec_helper'
 
 describe 'ntp::mac_os_x_client' do
-  let(:chef_run) { ChefSpec::SoloRunner.new(platform: 'mac_os_x', version: '10.11.1').converge('ntp::mac_os_x_client') }
+  let(:chef_run) do
+    ChefSpec::SoloRunner.new(
+      platform: 'mac_os_x',
+      version: '10.11.1'
+    ) do |node|
+      node.normal['ntp']['servers'] = %w(one two three)
```

Comment:
@scotthain  Yes, it has to be `normal` since `node.default` does not work unfortunately.